admin/webapp/websrc/app/frame/header/header.component.ts

```diff
@@ -165,13 +165,6 @@ export class HeaderComponent implements OnInit, OnDestroy {
           });
       });
 
-    if (!GlobalVariable.user?.remote_domain_permissions || !GlobalVariable.user?.extra_permissions) {
```

Comment:
What if use refreshToken function here instead of getSelf to fix this bug? Do we really need set remote permissions when user switch clusters?

Reply:
When Rancher SSO login, the user token doesn't contain the remote_global_permissions. The UI requires remote_global_permissions when switching to the managed cluster. This permission can only be retrieved by calling the /v1/selfuser API. Once the remote_global_permissions is set, the refreshToken function will not be called when switching to the managed cluster in the future.